### PR TITLE
Remove global list styles

### PR DESCRIPTION
**Pull request by mjbvz**
I'm trying to write a CSS rule that allows codicon in chat to render as normal. My previous `@layer` workaround caused a regression in other lists

The root problem I was trying to work around was that there's no way to clear rules that are set in the global list style sheet. You can override them with more specific css rules, but you can't remove them so that other global css rules (such as those for setting file icon colors) work properly

After discussion it sounds like we can try removing the global list styles entirely as they may not be used

### CODE REVIEW

**Review by roblourens — COMMENTED** [review]
Got this old issue- https://github.com/microsoft/vscode/issues/197574

It looks like we always apply the default styles to the individual list's style controller- https://github.com/microsoft/vscode/blob/0eb34edda80368e98762dacdc8740fdc2efc75b4/src/vs/platform/list/browser/listService.ts#L385 so maybe the global one is totally redundant?

File: src/vs/platform/list/browser/listService.ts [hunk 1]
```diff
@@ -490,6 +490,8 @@ export class WorkbenchPagedList<T> extends PagedList<T> {
 			this.updateStyles(options.overrideStyles);
 		}
 
+		this.getHTMLElement().classList.toggle('default-styled-list', !options.overrideStyles);
```
Comment (roblourens) [inline src/vs/platform/list/browser/listService.ts:493]: But if I only set one color in overrideStyles, then do I lose all the default colors?

**Comment by mjbvz** [comment]
Thanks @roblourens! Removing the global styles would be best and  it seems to work fine in quick testing. I was worried about breaking some part of the UI I hadn't tested tho, which is why I went with the current approach 

@aeschli  @joaomoreno What do you think about trying to remove the global styles entirely?

**Comment by roblourens** [comment]
I previously talked to Martin on Slack and Joao via https://github.com/microsoft/vscode/issues/197574 and I don't think anyone knows why we would still need the global styles. I think we should just look at the code carefully, delete it, and audit the styles on lists including ones that aren't WorkbenchList. I can spend a little time on this today

**Review by roblourens — DISMISSED** [review]

**Review by DonJayamanne — APPROVED** [review]

**Pull request merged**